jumpscale/tools/nginx/nginxserver.py

```diff
@@ -0,0 +1,30 @@
+from jumpscale.god import j
+from jumpscale.core.base import Base, fields
+
+
+class NginxServer(Base):
+    name = fields.String(default="main")
+    path = fields.String(default="~/sandbox/cfg/nginx/main/nginx.conf")
+
+    def install(self):
+        j.sals.process.execute("apt install nginx", showout=True, die=False)
+
```

Comment:
we can install certbots too https://certbot.eff.org/lets-encrypt/ubuntuxenial-nginx.html for ssl